### ISSUE TITLE
Can't Create or Delete User

### ISSUE THREAD

**Issue by johnaweiss** [GetSimpleCMS-CE/GetSimpleCMS-CE#7]
when i click the trash-bin next to a username, the Manage Users page refreshes, but the user doesn't get deleted.

**Comment by risingisland** [comment]
I am assuming you are using the user manager that is bundled with CE.
Did you refresh the page, to see if its a cache problem?
What version of CE and Massive Admin are you using?

**Comment by johnaweiss** [comment]
This is a new install of CE 
CE Version 3.3.19.1
Massive Admin Version 5.0.1
Refreshing User Manager page didn't fix it. The deleted user is still there.

**Comment by johnaweiss** [comment]
i tried to edit gsconfig using Massive plugin, and got this error:
```
Internal Server Error
The server encountered an internal error or misconfiguration and was unable to complete your request. Please contact the server administrator at webmaster@gsce.bayviewboom.org to inform them of the time this error occurred, and the actions you performed just before this error. More information about this error may be available in the server error log.
```
Then i enabled debug mode by editing gsconfig.php directly.
Then i tried to edit delete a user with Massive, and got the same behavior described above. I didn't get any debug error messages.  
Then i tried to edit gsconfig.php using Massive, and got the same behavior described above. I didn't get any debug error messages.

**Comment by risingisland** [comment]
You may be having an issue with ModSecurity, although it usually gives a different message.
Can you edit a template via the themes page?
If you get the same type of message, maybe try de-activating ModSecurity from cpanel.
In the meantime, I will forward this info off to @multicolor-rgb to see if he has any idea.

**Comment by johnaweiss** [comment]
> Can you edit a template via the themes page? 

yes.

> I will forward this info off to @multicolor-rgb to see if he has any idea.

thx!

**Comment by risingisland** [comment]
Maybe view Health Check in the Support page to see what the permissions for `/data/users/` are.
Multi will also take a look as soon as he can.

**Comment by johnaweiss** [comment]
just discovered i cannot create new users.

**Comment by johnaweiss** [comment]
On more testing, i see that Massive can create and delete users no problem, as long as no caps were used when the user was created.

**Comment by risingisland** [comment]
Thank you for the detective work. 
Multi is busy right now and hasnt had time, but I am guessing he will have a fix for this in a few days.

**Comment by risingisland** [comment]
Just saw your post on the forum too.
I didnt know you were trying to use MA user manager and a separate user plugin together.
MA will be updated to only allow lowercase.

**Comment by multicolor-rgb** [comment]
Hi,
Now i work with block possiblity Upper case user name. Looks like some hosting provider don't like name like this in GS. Fix soon

**Comment by risingisland** [comment]
@johnaweiss MA has now been updated to prevent using a capital letter in a new users name.

You can find the updated version here:
https://getsimple-ce.ovh/ce-plugins/
https://github.com/GetSimpleCMS-CE-plugins/plugin-Massive_Admin_Theme/tree/5.0.4
or via the "Download CE Plugins" via the plugins tab in you install.

**Comment by johnaweiss** [comment]
Thx for fix!
Do you "block" uppercase? If so, i think it would be smoother for user to simply _convert_ their entry to lowercase. 
I wasn't using two user-managers at the same time. I only activated one at a time.

**Comment by risingisland** [comment]
If you are typing in uppercase, it converts it to lower as you type.

**Comment by johnaweiss** [comment]
I'm still unable to delete users, even if no caps in the username.

**Comment by multicolor-rgb** [comment]
So maybe we must force chmod for every file what MA create? Soon check this.

**Comment by GetSimpleCMS-CE** [comment]
@johnaweiss Can you give the new version (5.0.6) a try? 
Chmod was added, so it should be fine now.

https://github.com/GetSimpleCMS-CE-plugins/plugin-Massive_Admin_Theme

**Comment by GetSimpleCMS-CE** [comment]
Yes, you should be able to overwrite your existing with new. Or, in plugins sidebar menu, choose from Download CE Plugins.

Yes, many hosts have their own settings, which can deffer from others. This is the first time, that I know of that someone is having this issue.

It is not going to delete existing users, as it does not set file permissions to existing files. Try to add and delete a new user.

**Comment by johnaweiss** [comment]
I created a new user with Massive, and then tried to delete it. It failed to delete the user.

**Comment by risingisland** [comment]
@johnaweiss Has this issue been resolved?
